Alicerce/Sources/Logging/Log+ConsoleLogDestination.swift

```diff
@@ -10,8 +10,8 @@ import Foundation
 
 public extension Log {
 
-    public class ConsoleLogDestination: LogDestination
-    {
+    public class ConsoleLogDestination: LogDestination {
+        
```

Comment:
Please turn on automatic whitespace trimming on Xcode to avoid unnecessary whitespace diffs in the future (not this case though) 😉